### PR TITLE
Add missing deployment and enviorment attributes

### PR DESCRIPTION
**Pull request by katrinSaleschus**

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/Environment.java
Comment (jmini) [inline src/main/java/org/gitlab4j/api/models/Environment.java:1]: Please update the corresponding json file (add a realistic example value): [environment.json](https://github.com/gitlab4j/gitlab4j-api/blob/c393f61a5aec8cb916da6b19b974ee67ba800979/src/test/resources/org/gitlab4j/api/environment.json)

This is tested by this test:
https://github.com/gitlab4j/gitlab4j-api/blob/c393f61a5aec8cb916da6b19b974ee67ba800979/src/test/java/org/gitlab4j/api/TestGitLabApiBeans.java#L171-L175

File: src/main/java/org/gitlab4j/api/webhook/DeploymentEvent.java
Comment (jmini) [inline src/main/java/org/gitlab4j/api/webhook/DeploymentEvent.java:1]: There is not test, but this should be tested in [`TestGitLabApiEvents`](https://github.com/gitlab4j/gitlab4j-api/blob/c393f61a5aec8cb916da6b19b974ee67ba800979/src/test/java/org/gitlab4j/api/TestGitLabApiEvents.java)

**Review by jmini — CHANGES_REQUESTED** [review]
Some tests should be added

**Comment by katrinSaleschus** [comment]
Thank you for your Review. Now everything should be ok to merge.

**Comment by jmini** [comment]
I will have a look

**Pull request merged**